pkg/vulnsrc/node/node.go

```diff
@@ -36,10 +37,64 @@ type RawAdvisory struct {
 	Overview           string
 	Recommendation     string
 	References         []string
-	CvssScoreNumber    json.Number `json:"cvss_score"`
+	CvssScoreNumber    interface{} `json:"cvss_score"`
 	CvssScore          float64
 }
 
+// This is for Go 1.14+ compat, to support mixed strings of CVSSScores
+// In Node core CVSSScore is like: "4.8 (Medium)", Type string
+// In NPM package CVSSScore is like: 4.8, Type float64
+// Details: https://github.com/golang/go/issues/37308
+func (ra *RawAdvisory) UnmarshalJSON(b []byte) error {
```

Comment:
You don't have to define `UnmarshalJSON` for `RawAdvisory`. All you have to do is define a new struct instead of json.Number.
https://play.golang.org/p/QZna0RK3gNF